apps/backend/package.json

```diff
@@ -4,8 +4,33 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "dev": "ts-node-dev --inspect --transpile-only --ignore-watch node_modules --respawn src/server.ts",
```

Comment:
issue: i think its better we use [tsx lib](https://www.npmjs.com/package/tsx) for that, it gets better performance and uses less resources from host machine

Reply:
I think we can talk about that, when we start the development.
Because there probably needs a specific setup at "dev" and your help will be helpful